components/style/Font.js

```diff
@@ -75,15 +75,9 @@ export default () => (
       /* latin */
       @font-face {
         font-family: 'IBM Plex Mono';
-        font-display: swap;
-        font-style: italic;
```

Comment:
Looks like all you actually need to remove is `font-style`.

Reply:
Ok, I already fixed this